build.xml

```diff
@@ -441,7 +441,10 @@
     </target>
 
     <target name="clean" description="Remove all locally created artifacts">
-        <delete dir="${build.test.dir}" />
+        <delete includeemptydirs="true" failonerror="false">
+            <fileset dir="${build.test.dir}/cassandra" includes="**/*"/>
+            <fileset dir="${build.test.dir}" includes="**/*" excludes="cassandra"/>
```

Comment:
why do we need *two* lines?
Is it to clean up everything under `${build.test.dir}` but retain the `/cassandra` subdir?